src/test/java/seedu/address/logic/parser/delivery/DeliveryListCommandParserTest.java

```diff
@@ -70,7 +75,35 @@ public void parse_validArgs_returnsDeliveryListCommand() {
         CommandParserTestUtil.assertParseSuccess(parser, VALID_DELIVERY_LIST_SORT_DESC,
             new DeliveryListCommand(null, null, null, Sort.DESC));
 
+    }
+
+    @Test
+    public void parse_duplicateArgs_returnsDeliveryListCommand() {
+        CommandParserTestUtil.assertParseSuccess(parser, VALID_DELIVERY_LIST_SHIPPED + VALID_DELIVERY_LIST_CANCELLED,
+            new DeliveryListCommand(DeliveryStatus.CANCELLED, null, null, Sort.DESC));
+        CommandParserTestUtil.assertParseSuccess(parser,
+            VALID_DELIVERY_LIST_DELIVERY_DATE + VALID_DELIVERY_LIST_DELIVERY_DATE,
+            new DeliveryListCommand(null, null, new DeliveryDate(VALID_DELIVERY_DATE_3), Sort.DESC));
```

Comment:
Would this be better if they used the same constant? e.g. `VALID_DELIVERY_DATE_3`  or `VALID_DELIVERY_LIST_DELIVERY_DATE`

Reply:
One contains a prefix while the other is just a pure date.